app/jobs/send_purge_warnings_job.rb

```diff
@@ -8,15 +8,12 @@ def perform(*_args)
     sent = []
     not_sent = []
 
-    # Get admin users
-    recipients = User.where(role: [Roles::ADMIN, Roles::SUPER_USER])
+    # Get admin users but skip USA admins and locked users
+    recipients = User.where(role: [Roles::ADMIN, Roles::SUPER_USER], locked_at: nil).where.not(jurisdiction: 'USA')
```

Comment:
`where.not(jurisdiction: 'USA')` is not a valid check as it expects an ID or jurisdiction rather than just a string name. 
You can test it locally in your console and notice it gives nothing back.

This should be `where.not(jurisdiction_id: Jurisdiction.find_by(name: 'USA').id)`